.github/ISSUE_TEMPLATE/bug-report.yml

```diff
@@ -40,15 +40,15 @@ body:
     required: true
 - type: input
   attributes:
-    label: What jax/jaxlib version are you using?
+    label: What jax/jaxlib version are you using? Include the output of `import jax, jaxlib; print(jax.__version__, jaxlib.__version__)`
```

Comment:
Unfortunately code rendering doesn't work in this context; you can click "view file" in the github PR to see a preview. Maybe try adding this info in a `description:` field?

Reply:
Done.